app/posts/smoking-cessation/2025/03/2025-03-20-smoking-cessation-pilot.md

```diff
@@ -0,0 +1,143 @@
+---
+title: "Smoking cessation pilot"
+description:
+  "Summary of work done to support the launch of a smoking cessation support service."
+date: 2025-03-20
+tags:
+---
+
+To achieve the government goal of a smoke-free Britain by 2030, populations with increasing dependence need access to smoking treatment services which meets their needs.
+
+Currently, people who want to stop smoking tobacco can use:
+- in-person NHSE services via routes like their pharmacy or GP
+- digital apps
+- work out a plan on their own
+
+People who want more guided support but cannot access in-person services for any reason do not have an NHSE digital service to use.
+
+Availability of tobacco treatment services is not consistent across the country and does not reach populations with increasing tobacco dependency (younger members of the population, men, and those from more deprived backgrounds).
+
+A third of surveyed local authorities (33%) did not commission a universal service to help people stop smoking in 2022 (latest available data).
+
+Local Authority tobacco treatment services are mostly delivered face-to-face, which is harder for people who work shifts or travel for their work to access. These services are mostly accessed by the middle-to-older age groups, and fewer men accessing from younger age groups than women.
+
+Evidence from the national digital weight management service shows that digital services can engage more deprived populations and fill gaps in provision. This evidence comes from increased access and engagement rates across all indices of multiple deprivation (IMD) bands (measure of relative deprivation for small, fixed geographic areas of the UK).
+
+## Aim
+
+To create a digital pathway for tobacco cessation that helps people quit smoking while accessing the support they need, with a particular focus on those currently underserved by in-person services.
```

Comment:
I think the same here as above. It might be hard to detail, but it's hard to grasp what this brings to the party?